earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -85,14 +104,19 @@ void validateXMLParameters()
     if (initialDOMHeapAllocSize < lowestBlock ||
    	maxDOMHeapAllocSize < lowestBlock)     
     {
-	throw MinValuesNotMet();
+        throw MinValuesNotMet();
     }
 
     // check to make sure that the initial size is less than the max size
     if (maxDOMHeapAllocSize < initialDOMHeapAllocSize ||
         maxDOMHeapAllocSize < maxDOMSubAllocationSize)
     {
         throw SizeError();
+    }
+
+    if (percent > 5 || percent < 1)
```

Comment:
nitpick: comparing float against ints and magic numbers.  Could make constants and then re-use in PercentError